src/Background.cpp

```diff
@@ -326,27 +328,22 @@ namespace Background {
 							// use a logarithmic scala for brightness since this looks more natural to the human eye
 							float brightness = log(luminositySystemSum / (4 * M_PI * distance.Length() * distance.Length()));
 
-							stars.pos.push_back(distance.Normalized() * 1000.0f);
+							// galaxy is Z-up but the scene is Y-up
```

Comment:
I think it'd be simpler to handle this in Z-up space and merely set the transform matrix to rotate Y-up to Z-up when rendering the starfield.

Reply:
Done! Yes, it’s better this way, considering that in the possible future I would like each system to have its own transformation matrix. (It is strange that the axes of all our systems are parallel to the axis of the galaxy.)